include/dump_utils.hpp

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include "async_resp.hpp"
+#include "error_messages.hpp"
+#include "logging.hpp"
+#include "utility.hpp"
+
+#include <boost/algorithm/string/case_conv.hpp>
+#include <boost/system/error_code.hpp>
+#include <dbus_singleton.hpp>
+#include <dbus_utility.hpp>
+
+#include <cstddef>
```

Comment:
why this c header file nedde? use c++ header file instead

Reply:
It defines `crow::connections::systemBus` which is used in `dump_utils.hpp`.